otj-kafka-spring/src/main/java/com/opentable/kafka/spring/builders/SpringKafkaConsumerFactoryBuilder.java

```diff
@@ -27,6 +27,7 @@
 import org.springframework.kafka.core.ConsumerFactory;
 import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
 import org.springframework.kafka.listener.BatchErrorHandler;
+import org.springframework.kafka.listener.CommonErrorHandler;
```

Comment:
@rushalias have you compared all our current settings and defaults, which may have changed in Kafka 3 (in fact I'm pretty sure I read some did)

Reply:
this has not been done, I concentrated getting the build to build.